.github/workflows/workflow-vercel-deployment.yml

```diff
@@ -1,11 +1,22 @@
 name: Vercel Preview Deployment
 
+on:
+  workflow_call:
+    inputs:
+      vercel-project-name:
+        required: true
+        type: string
+      deployment_environment:
```

Comment:
using underscore here but dashes in the other input parameter

naming conventions for the workflow should be added to our naming ADR